pkg/image/controller/controller.go

```diff
@@ -87,9 +87,10 @@ func (c *ImportController) Next(stream *api.ImageStream) error {
 }
 
 // getTags returns tags from default upstream image repository and explicitly defined.
-// Returns a map of tags to be imported and an error if one occurs.
+// Returns a map of tags to be imported, an error if one occurs and flag saying if
+// we should retry imports, meaning not setting the import annotation.
 // Tags explicitly defined will overwrite those from default upstream image repository.
-func getTags(stream *api.ImageStream, client dockerregistry.Client, insecure bool) (map[string]api.DockerImageReference, error) {
+func getTags(stream *api.ImageStream, client dockerregistry.Client, insecure bool) (map[string]api.DockerImageReference, error, bool) {
```

Comment:
Change the signatures of all of these methods so that error is last (error is always last)